Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -13,6 +13,8 @@ class BraveWebView: WKWebView {
             configuration.websiteDataStore = WKWebsiteDataStore.nonPersistent()
         }
         
+        // Enables Zoom in website by ignoring their javascript based viewport Scale limits.
+        configuration.ignoresViewportScaleLimits = true
```

Comment:
Should probably put this with all the other configuration edits (for easier finding it later):

https://github.com/brave/brave-ios/blob/development/Client/Frontend/Browser/Tab.swift#L188-L191

Though I realize `TabWebView` subclasses `BraveWebView` (I don't think we need both, maybe we should bring `BraveWebView` stuff over to `TabWebView` and delete it)

Reply:
Thanks! Was thinking where to put it myself. :)